pkg/sanity/sanity.go

```diff
@@ -357,6 +354,22 @@ func loadSecrets(path string) (*CSISecrets, error) {
 	return &creds, nil
 }
 
+// loadMap reads a map from given file path.
+func loadMap(path string) map[string]string {
```

Comment:
I think you can make this more generic:
```
func loadFromFile(from string, to interface{}) {
    if len(path) != 0 {
...
        err = yaml.Unmarshal(yamlFile, to)
    }
}
```

Then call it with:
```
loadFromFile(reqConfig.TestSnapshotParametersFile, &reqConfig.TestSnapshotParameters)
```

Untested, but I think it'll work, and then we have something that could also become useful for populating other data structures.

Reply:
OK, I will update and test it.

Reply:
Hi @pohly , The method is worked and I updated this PR. Would you please review again?